.travis.yml

```diff
@@ -1,4 +1,25 @@
 language: ruby
+
+sudo: false
+
+cache: bundler
+
 rvm:
-  - 1.9.3
-before_install: gem install bundler -v 1.11.2
+  - 2.0.0
+  - 2.1
+  - 2.2
+  - 2.3.0
+  - ruby-head
```

Comment:
Ah I see it now. Sorry, newish to actually using Travis.

On Tue, Aug 9, 2016 at 3:37 PM, Peter Souter <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/pizzaops/rubocop-facter/pull/2#discussion_r74157593>:
>
> >  rvm:
> > -  - 1.9.3
> > -before_install: gem install bundler -v 1.11.2
> > +  - 2.0.0
> > +  - 2.1
> > +  - 2.2
> > +  - 2.3.0
> > +  - ruby-head
>
> We test against ruby-head but alllow failures, it's a canary technique
> that allows you to allow for future proofing without breaking everything
> 👍
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/pizzaops/rubocop-facter/pull/2/files/091cec6e4ece0fe29ba6aae12b7afbd24591dbfd#r74157593>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAfHm6LVYmfhrChpgQrfmyggIxERCjaiks5qeQE5gaJpZM4Ijpof>
> .
>